mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/135 - component software tab

Closes #135 

To test:
- Run any scenario
- Open ComponentView tab and select computer
- Go to Software tab and make sure installed software is in correct list
- Install software and make sure funds are reduced by software's install cost
- Uninstall software and make sure it goes to available software list
